fe/fe-core/src/main/java/org/apache/doris/nereids/types/TimeV2Type.java

```diff
@@ -57,6 +64,28 @@ public boolean equals(Object o) {
         return o instanceof TimeV2Type;
     }
 
+    //TODO: implement this method by parsing the string and get the scale of it
+    public static TimeV2Type forTypeFromString(String s) {
+        return MAX;
```

Comment:
you should implement this function, because the result will different after implement it. 

for example
```sql
select time '11:12:13.123456' = time '11:12:13.12' -- should return false
```

so you should better implement this function in this pr

Reply:
I've done this. but I dont think it will make `select time '11:12:13.123456' = time '11:12:13.12'` return false. bacause:
```sql
mysql> select cast("2020-12-12 12:12:12.123" as datetime) = cast("2020-12-12 12:12:12.1234" as datetime);
+--------------------------------------------------------------------------------------------+
| cast("2020-12-12 12:12:12.123" as datetime) = cast("2020-12-12 12:12:12.1234" as datetime) |
+--------------------------------------------------------------------------------------------+
|                                                                                          1 |
+--------------------------------------------------------------------------------------------+
```
for anytype without scale, we dont its length now but just use zero